### ISSUE TITLE
How to make Energy? 

### ISSUE THREAD

**Issue by MaHoE-github** [Lothrazar/Cyclic#1240]
Minecraft Version: 1.12.2

Forge Version: 14.23.5.2847

Mod Version:  1.15.7

Single Player or Server: Both

Describe problem (what you were doing; what happened; what should have happened):

Crafted very heavy and strong equipment (i.e obsidian armor), noticed that it had some distastful enchantments already on it, decided to make a void anvil to remove aforementioned enchantments but It requires energy to remove. Decided to research on how to make energy and only known real method (without using cheats/creative) is by making peat, to make peat need biomass. Biomass needs a Hydrator to be made, Hydrator requires energy to work.

TL;DR : I dont know how to make energy without using peats which in my patch require energy to make.

Log file link:

Video/images/gifs (direct upload or link): 

If your bug is related to anything causing LAG you need to measure the lag by taking a sample https://minecraft.curseforge.com/projects/sampler

**Comment by Lothrazar** [comment]
what other mods do you have installed? 

You should not need peat for energy, just craft the biomass and peat bog blocks. place near water. harvest when they turn green.  By then when we have some of these the generator should be craftable.

Which step is having a problem

**Comment by MaHoE-github** [comment]
We do have some other mods added, but most dont interfere with each other and are other biome mods, The Betweenlands mod for eg 
![Screenshot (27)](https://user-images.githubusercontent.com/55921764/65819478-a7a37a00-e23a-11e9-8a43-02e0b40bcdaf.png)

This is what shows up when I search for Biomass to find the recipe, it apparently needs a Hydrator to craft. And the Hydrator also needs energy to work.
![Screenshot (28)](https://user-images.githubusercontent.com/55921764/65819515-37492880-e23b-11e9-9644-493e6fb223f6.png)

**Comment by Lothrazar** [comment]
Thanks ill check

**Comment by Lothrazar** [comment]
Yes this issue was addressed, new texture as well
![peat bog](https://user-images.githubusercontent.com/621529/65820992-9f205480-e1e4-11e9-8275-756c306cfb56.PNG)

**Comment by Lothrazar** [comment]
![1111](https://user-images.githubusercontent.com/621529/65820993-bb23f600-e1e4-11e9-8944-1b7dbcd8ce90.PNG)

**Comment by MaHoE-github** [comment]
I see, so I need to update the mod then?
